### PR TITLE
Change log location to a fixed folder

### LINKED ISSUE #378: Log files end up in random paths
Please, don't do this. As this plugin is only used in Seravo environment, maybe just define log directory somewhere, eg. `define('SERAVO_LOG_DIRECTORY', '/data/log');` and then use that as basis for setting up correct path. Now logfiles can be found from lots of parent directories if someone executes WP CLI in child directory.

https://github.com/Seravo/seravo-plugin/blob/7944fcb01598b5502e332e760c73920545b0e379/modules/wp-plugin-log.php#L80-L86

Example: If someone executes WP CLI in home directory, `/home/wp-settings.log` appears.

### PR DESCRIPTION
**Pull request by Wiljami**
Logs were ending up in random places.

This fix sets a fixed location for the logs to '/data/log/' folder.

Closes #378

### CODE REVIEW

**Comment by Wiljami** [comment]
Fixed the slashes

File: modules/wp-login-log.php [hunk 1]
```diff
@@ -64,12 +64,8 @@ public static function wp_login_redirect_log( $redirect_to, $requested_redirect_
       $http_user_agent = isset($_SERVER['HTTP_USER_AGENT']) ? $_SERVER['HTTP_USER_AGENT'] : '';
 
       // Finally write the log to disk
-      $log_directory = dirname(ini_get('error_log'));
-      if ( empty($log_directory) ) {
-        // If there is no log directory, just log one directory above WordPress
-        // and hope that directory is writeable but not accessible from the web
-        $log_directory = '..';
-      }
+      $log_directory = '/data/log';
```
Comment (ypcs) [inline modules/wp-login-log.php:67]: If this log path is defined as variable, I think you can just merge it to filename in the next line (`$log_fp = ...`), no need to have separate variable if we use it only once.

File: modules/wp-plugin-log.php [hunk 2]
```diff
@@ -78,13 +78,8 @@ public static function on_switch_theme( $theme ) {
     }
 
     public static function write_log( $message ) {
-      $log_directory = dirname(ini_get('error_log'));
+      $log_directory = '/data/log';
```
Comment (ypcs) [inline modules/wp-plugin-log.php:81]: See another comment.

**Comment by Wiljami** [comment]
Made changes ypcs suggested.

File: modules/wp-login-log.php [hunk 3]
```diff
@@ -64,13 +64,8 @@ public static function wp_login_redirect_log( $redirect_to, $requested_redirect_
...
       // Finally write the log to disk
-      $log_directory = dirname(ini_get('error_log'));
-      if ( empty($log_directory) ) {
-        // If there is no log directory, just log one directory above WordPress
-        // and hope that directory is writeable but not accessible from the web
-        $log_directory = '..';
-      }
-      $log_fp = fopen($log_directory . '/wp-login.log', 'a');
+
+      $log_fp = fopen($log_directory . '/data/log/wp-login.log', 'a');
```
Comment (elguitar) [inline modules/wp-login-log.php:68]: It seems to me that `$log_directory` is not defined before this.

File: modules/wp-plugin-log.php [hunk 4]
```diff
@@ -78,20 +78,13 @@ public static function on_switch_theme( $theme ) {
...
-        $log_directory = '..';
-      }
       $time_local = gmdate('j/M/Y:H:i:s O');
 
       $current_user = wp_get_current_user();
 
       $username = $current_user->user_login;
 
-      $log_fp = fopen($log_directory . '/wp-settings.log', 'a');
+      $log_fp = fopen($log_directory . '/data/log/wp-settings.log', 'a');
```
Comment (elguitar) [inline modules/wp-plugin-log.php:87]: It seems to me that $log_directory is not defined before this.

Comment (Wiljami) [reply modules/wp-plugin-log.php]: A brainfart. Removed it as it does nothing.

**Comment by elguitar** [comment]
Something shady is happening with the CI tests. @ypcs do you have any idea? I found this: https://github.community/t/github-actions-pull-requests-always-include-some-outdated-checks/16157
I have no rights to access github actions and branch protection settings.

**Comment by ottok** [comment]
It is Travis-CI, not Github Actions. Something weird is on the link from this PR to Travis-CI, but you can see it at https://travis-ci.org/github/seravo/seravo-plugin/pull_requests

Tests passed at
https://travis-ci.org/github/Seravo/seravo-plugin/builds/706128731

Try rebasing on re-pushing to trigger Github to update the status.

**Comment by elguitar** [comment]
@Wiljami Please rebase on master to start the Travis CI.

**Pull request merged**